### DISCUSSION TITLE
use this template button missing github how to enable it

### DISCUSSION THREAD

**Discussion by bhupesh7shakya** [community/community#130131] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

![image](https://github.com/community/community/assets/54110001/f2a1724c-9afa-49c5-ba56-f5acf5d5104a)
i have enable the template but

**Comment by mantolic** [comment, marked as answer]
Looks like button is in dom, but it's hidden. I found workaround, it is possible to remove `display: none !important` and the button will be visible and usable, it's possible to use the template :) 
![image](https://github.com/community/community/assets/7883663/23ec2e25-ed14-4bb8-88fb-f1cc965016fd)
![image](https://github.com/community/community/assets/7883663/f79ce7c9-4e8d-4e3f-81b2-814859be70e6)

**Reply by bhupesh7shakya** [reply]
thank you bro they updated

**Comment by gitstua** [comment]
You should be able to enable a repo to be a template using the instructions at https://docs.github.com/en/enterprise-cloud@latest/repositories/creating-and-managing-repositories/creating-a-template-repository#creating-a-template-repository

**Reply by ghbm-itk** [reply]
That's literally what he's showing in the first image. Don't answer if you doin't understand the question

**Comment by TyronNIKO** [comment]
**Use this template** has returned! I think problem solved...

**Reply by bhupesh7shakya** [reply]
yes it has return

**Comment by paleskincodes** [comment]
"use this template" is also missing from my github dash, which since Im a newb, its hard for me to use others' repos as templates

can someone help me

**Comment by gowthamprasath14** [comment]
good

**Comment by RafaelAngeloPiad** [comment]
missing now im using google chrome may 19, 2025
